module/module_test.go

```diff
@@ -146,6 +146,7 @@ func TestExampleService(t *testing.T) {
 	// TODO: this doesn't yet generate the build to a dir
 	// TODO: this should return a collection of errors if they occur
 	err = moduleSystem.GenerateBuild(
+		"github.com/uber/zanzibar",
```

Comment:
Not really. It doesn't matter because as the comment says, we're not actually generating a build in this test. It doesn't matter until we're actually using it to generate code. The package name would be the contents of the example service being generated for the test case (which doesn't exist).